spec/models/users_spec.rb

```diff
@@ -20,7 +20,7 @@
   end
```

Comment:
Let's add specs for the content types if not added previously

Reply:
it { is_expected.to validate_content_type_of(:profile_photo).allowing('image/jpg', 'image/jpeg', 'image/png') }